vars.md

```diff
@@ -11,19 +11,19 @@ assured that they are persisted.
 | :------ | :------- | :--------------- | :-------- |
 | EPHEMERAL_CLUSTER | Tool for running management/ephemeral cluster. | minikube, kind, tilt | Ubuntu default is kind. Only minikube is supported on CentOS |
 | EXTERNAL_SUBNET                | This is the subnet used on the "baremetal" libvirt network, created as the primary network interface for the virtual bare metalhosts.                                                                                                                    | CIDR                               | 192.168.111.0/24                                             |
-| SSH_PUB_KEY                    | This SSH key will be automatically injected into the provisioned host by the provision_host.sh script.                                                                                                                                                   |                           | ~/.ssh/id_rsa.pub                                            |
+| SSH_PUB_KEY                    | This SSH key will be automatically injected into the provisioned host by the lib/common.sh script.                                                                                                                                                   |                           | ~/.ssh/id_rsa.pub                                            |
```

Comment:
Could you please refer to code location where this happens?

Reply:
Sure, [here](https://github.com/metal3-io/metal3-dev-env/blob/master/lib/common.sh#L60) it is

Reply:
I think we declare the key in `lib/common.sh`, but this key will be automatically injected into the provisioned host by another script, no?

Reply:
Which script do you mean by another script?

Reply:
Current `01_prepare_host.sh` is what `provision_host.sh` had been in my opinion, is that what you meant?

Reply:
No, I mean the script which does something like `ssh-add <key><host>`. If I am not mistaken, script 02 or 03 do that.

Reply:
You were right, I have changed it, thanks for review!